runtime/gc_glue_java/RealtimeGCDelegate.cpp

```diff
@@ -132,7 +132,7 @@ MM_RealtimeGCDelegate::allocateAndInitializeUnfinalizedObjectLists(MM_Environmen
 bool
 MM_RealtimeGCDelegate::allocateAndInitializeOwnableSynchronizerObjectLists(MM_EnvironmentBase *env)
 {
-	const UDATA listCount = MM_HeapRegionDescriptorRealtime::getOwnableSynchronizerObjectListCount(env);
+	const UDATA listCount = MM_RealtimeGCDelegate::getOwnableSynchronizerObjectListCount(env);
```

Comment:
There is no need to specify `MM_RealtimeGCDelegate::` here. This is a member of this class